neutrino-preset.js

```diff
@@ -0,0 +1,207 @@
+'use strict';
```

Comment:
I think this configuration is pretty good for the current Neutrino. Neutrino v4 should clean this up quite a bit.

My one recommendation would probably be to break this up into a few files, in a directory named `neutrino-preset`. Then your preset would just be the same, `neutrino-preset`.